CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+20.0.0
+======
+- Added support for version 16.1 of the Google Ads API.
+- Updated code examples:
+  * VerifyAdvertiserIdentity
+  * UploadConversionAdjustment
+- Removed obsolete code examples:
+  * AddFlightsFeed
+  * AddRealEstateFeed
+  * ApproveMerchantCenterLink
+  * RejectMerchantCenterLink
```

Comment:
nit: Also mention removal of RemoveFlightsFeedItemAttributeValue and UpdateFlightsFeedItemStringAttributeValue

Reply:
Done.